hyperstart.go

```diff
@@ -422,7 +425,7 @@ func (h *hyper) startOneContainer(pod Pod, c Container) error {
 		Process: process,
 	}
 
-	if err := h.bindMountContainerRootfs(pod.id, c.id, c.rootFs, c.config.ReadonlyRootfs); err != nil {
+	if err := h.bindMountContainerRootfs(pod.id, c.id, c.rootFs, false); err != nil {
```

Comment:
Because hyperstart actually does not support this to be set to true. Docker does not set this to true, but I noticed that CRI-O does that, and we end up with hyperstart failure when it tries to do `mkdir /lib/modules`. This should be fixed inside hyperstart before we can re-enable this.